test/f-promise-test.ts

```diff
@@ -124,4 +124,26 @@ describe(module.id, () => {
         strictEqual(queue.read(), 9);
         strictEqual(queue.peek(), undefined);
     });
+
+    describe('collection functions', (done) => {
+        it('map', () => {
+            run(() => {
+                const fn = (encoding) => {
+                    const fname = fsp.join(__dirname, '../../test/f-promise-test.ts');
+                    const length = wait(mzfs.readFile(fname, encoding)).length;
+                    return length;
+                };
```

Comment:
I would simplify the test and just use the `delay` function:

`deepEqual(map([2, 5], delay), [2, 5])`